src/verifact_manager.py

```
@@ -38,90 +40,114 @@
-        """Process text through the full factchecking pipeline.
-
-        Args:
-            text: The text to factcheck
-            progress_callback: Optional function to call with progress messages
-            progress_msg: The Chainlit message object to update
-
-        Returns:
-            List[Verdict]: A list of verdicts for claims in the text
-        """
+    # Replace the run method with this one
+    async def run(self, query: str, progress_callback=None) -> list:
```

Ruff (0.11.9)

Missing docstring in public method

(D102)